rust/osm2lanes/src/lib.rs

```diff
@@ -23,6 +23,27 @@ pub struct Road {
     pub lanes: Vec<Lane>,
 }
 
+impl Road {
+    pub fn has_separators(&self) -> bool {
+        self.lanes.iter().any(|lane| lane.is_separator())
+    }
+}
+
+impl Road {
+    pub fn total_width(&self) -> Metre {
+        self.lanes
+            .iter()
+            .map(|lane| match lane {
+                Lane::Separator { markings } => markings
+                    .iter()
+                    .map(|marking| marking.width.unwrap_or(Metre(0.2)))
```

Comment:
Hmm you hardcoded this default in the web viewer too. How about `marking.get_width() -> Metre` that fills in the default?

Reply:
indeed. also panics everyhwere that I need to remove.